src/codemagic/tools/_app_store_connect/arguments.py

```diff
@@ -315,6 +316,36 @@ class BuildArgument(cli.Argument):
         ),
         argparse_kwargs={'required': False},
     )
+    LOCALE = cli.ArgumentProperties(
+        key='locale',
+        flags=('--locale',),
+        type=Locale,
+        description='The specified locale, e.g. de-DE or en-GB',
+        argparse_kwargs={
+            'required': True,
+            'choices': list(Locale),
+        },
+    )
+    LOCALE_OPTIONAL = cli.ArgumentProperties(
+        key='locale',
+        flags=('--locale',),
+        type=Locale,
+        description='The specified locale, e.g. de-DE or en-GB',
+        argparse_kwargs={
+            'required': False,
+            'choices': list(Locale),
+        },
+    )
+    WHATS_NEW = cli.ArgumentProperties(
+        key='whats_new',
+        flags=('--whats-new',),
+        type=str,
```

Comment:
As notes can be quite long and you'd input them probably from some file I suggest that you  create new type for it that extends `cli.EnvironmentArgumentValue[str]`. See `AppStoreConnectArgument.PRIVATE_KEY` for example.